libs/game/sprite.ts

```diff
@@ -259,6 +333,15 @@ class Sprite extends sprites.BaseSprite {
     setImage(img: Image) {
         if (!img) return; // don't break the sprite
         this._image = img;
+        this.recalcSize();
```

Comment:
might be worth wrapping in a img !== this._image check now since we're doing a decent amount of work in the case where the image is being trivially reset